src/mlpack/core/data/feature_selection.hpp

```diff
@@ -0,0 +1,57 @@
+/**
+ * @file feature_selection.hpp
+ * @author Jeffin Sam
+ *
+ * Feature selction based on variance thresholding.
+ * Motivated by the idea that low variance features contain less
+ * information.
+ * Calculate varience of each feature, then drop features 
+ * with variance below some threshold
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef MLPACK_CORE_DATA_FEATURE_SELECTION_HPP
+#define MLPACK_CORE_DATA_FEATURE_SELECTION_HPP
+
+#include <mlpack/prereqs.hpp>
+
+namespace mlpack {
+namespace data {
```

Comment:
I think if you are planing to add more selection methods, perhaps it makes sense to use an extra namespace like ``selection``. Let me know what you think.